.github/workflows/release-cli.yaml

```diff
@@ -115,17 +115,10 @@ jobs:
       matrix:
         include:
           - os: ubuntu-20.04
-            shell: bash
-          - os: macos-11
-            shell: bash
-          - os: m1
-            # TODO: Can we remove this if we upgrade our m1 runner to native arm64?
-            shell: /usr/bin/arch -arch arm64e bash --noprofile --norc -eo pipefail {0}
+          - os: macos-13
+          - os: macos-13-xlarge
```

Comment:
maybe add a comment here that one of these is x86_64 and one is arm64

Reply:
done